src/cljs/zetawar/data.cljs

```diff
@@ -18,7 +18,6 @@
 
     ;; Faction configuration
     :configure-faction-tip"Configure faction"
```

Comment:
Not your doing afaik, but we should add a space between the keyword and the string.